controllers/controllers.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+  // "encoding/json"
+  "net/http"
+  "io/ioutil"
+  "fmt"
+
+  "sebatibot/logger"
+)
+
+func Index(w http.ResponseWriter, r *http.Request) {
+  fmt.Fprint(w, "Welcome!\n")
+}
+
+func TelegramUpdate(w http.ResponseWriter, r *http.Request) {
+  log := logger.GetLogger()
+  
+  bodyBytes, _ := ioutil.ReadAll(r.Body)
+  bodyString := string(bodyBytes)
+
+  log.Debugf("Request Body: %s", bodyString)
+
+  // Write to db here
```

Comment:
Just a question: this really writes to db?

Reply:
No. Bot logic (message parsing, bot reply) will be located here.